src/ranged.cpp

```diff
@@ -3646,62 +3631,51 @@ void target_ui::draw_controls_list( int text_y )
 
     // Compile full list
     if( shifting_view ) {
-        lines.push_back( {8, colored( col_move, _( "Shift view with directional keys" ) )} );
+        lines.push_back( {8, colorize( _( "Shift view with directional keys" ), input_context::get_hint_color( move_hint_state ) )} );
     } else {
-        lines.push_back( {8, colored( col_move, _( "Move cursor with directional keys" ) )} );
+        lines.push_back( {8, colorize( _( "Move cursor with directional keys" ), input_context::get_hint_color( move_hint_state ) ) } );
     }
     if( is_mouse_enabled() ) {
         std::string move = _( "Mouse: LMB: Target, Wheel: Cycle," );
         std::string fire = _( "RMB: Fire" );
```

Comment:
Bound to actions

I changed the code to this.
```
std::string move = string_format( _( "%s %s" ), ctxt.get_hint_mouse( "SELECT", _( "Target" ),
                                        move_hint_state ), ctxt.get_hint_mouse( "SCROLL_UP", _( "Cycle" ), move_hint_state ) );
        std::string fire = ctxt.get_hint_mouse( "FIRE", fire_hint_state );
```

and implemented the `get_hint_mouse` function in case we want to use them elsewhere eventually